crate/pages/blog/Snowsprint-2015.html

```diff
@@ -0,0 +1,75 @@
+title: Crate.IO Snowsprint 2015 recap: Dockerzeit
+link: https://crate.io/blog/Snowsprint-2015/
```

Comment:
URLs must be slugs (lower case, no spaces, dashes instead, special chars are removed)